packages/vitest/src/node/plugins/index.ts

```diff
@@ -229,7 +232,7 @@ export async function VitestPlugin(options: UserConfig = {}, ctx = new Vitest('t
       async configureServer(server) {
         try {
           await ctx.setServer(options, server)
-          if (options.api && options.watch)
+          if (options.api && (options.watch || options.browser))
```

Comment:
I see `options.watch || options.browser` again, shouldn't it be inside a single check?

Reply:
sure.